projects/plugins/search/readme.txt

```diff
@@ -1,10 +1,10 @@
 === Jetpack Search ===
-Contributors: automattic, annamcphee, bluefuton, kangzj, jsnmoon
+Contributors: automattic, annamcphee, bluefuton, kangzj, jsnmoon, robfelty, gibrown 
 Tags: search, jetpack
```

Comment:
Remove "jetpack" and add some tags: "filter, woocommerce search, ajax search, product search"